pytorch_lightning/trainer/callback_config_mixin.py

```diff
@@ -12,14 +14,15 @@ def configure_checkpoint_callback(self):
         """
         if self.checkpoint_callback is True:
             # init a default one
-            if isinstance(self.logger, TestTubeLogger):
-                ckpt_path = '{}/{}/version_{}/{}'.format(
+            if self.logger is not None:
+                ckpt_path = os.path.join(
                     self.default_save_path,
-                    self.logger.experiment.name,
-                    self.logger.experiment.version,
-                    'checkpoints')
+                    self.logger.name,
+                    f'version_{self.logger.version}',
+                    "checkpoints"
+                )
             else:
-                ckpt_path = self.default_save_path
+                ckpt_path = "checkpoints"
```

Comment:
How about `os.path.join(self.default_save_path, "checkpoints")`? The point of this PR was to make sure that we're not storing checkpoints in the current directory, because the whole directory gets wiped out.

Reply:
Fixed!